packages/design-tokens/styles/base.css

```
@@ -122,6 +122,12 @@ mark::after {
   width: 1px;
 }
 
+label {
+  font-size: 0.875rem;
+  font-variation-settings: "'wght' 700";
```

This shouldn't be quoted, right?

```suggestion
  font-variation-settings: 'wght' 700;
```